.gitignore

```diff
@@ -15,5 +15,8 @@ lib/
 .classpath
 .settings
 
+#Atom
+tsconfig.json
```

Comment:
In hawkular-charts and hawkular, its actually required for tsfmt (which uses tslint.json and tsconfig.json) to work.